runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/internal/registry/ConfigureTask.java

```diff
@@ -160,16 +160,14 @@ else if ("http".equals(configURL.getProtocol()) || "https".equals(configURL.getP
             }
         }
 
-        if (config.configSyntaxMustache())
+        if (config.configResolveExpressions())
         {
-            ExpressionResolver expressionResolver = ExpressionResolver.instantiate();
-            for (String context: expressionResolver.names())
-            {
-                configText = expressionResolver.resolve(context, configText);
-            }
+            ExpressionResolver resolver = ExpressionResolver.instantiate();
```

Comment:
Let's move this instantiation to the constructor, so it is done once, even when detecting changed configuration (later).

Suggest calling it `expressions` so the code reads as `expressions.resolve(...)`.